root/components/home/About.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faCheckCircle} from '@fortawesome/free-solid-svg-icons';
+
+const About = (): React.Element<'section'> => {
+  return (
+    <section className="about">
+      <div className="container">
+        <div className="row">
+          <div
+            className="col-lg-6"
+          >
+            <h3
+              className="heading"
+            >
+              {l('About')}
+              {' '}
+              <span
+                className="color-purple"
+              >
+                {l('Music')}
+              </span>
+              <span
+                className="color-orange"
+              >
+                {l('Brainz')}
+              </span>
+            </h3>
+            <p>
+              {l('MusicBrainz is an open music encyclopedia ')}
+              {l('that collects music metadata and makes ')}
+              {l('it available to the public.')}
```

Comment:
These three lines should be one l() block, since it's all one sentence and it should be translated as such. Otherwise, people will have to translate "it available to the public" on its own and whatnot :) Same for the other sets of lines below. I'll amend.

Reply:
Thanks to you for fixing it :)